charts/illumidesk/templates/grader_setup_service.yaml

```diff
@@ -101,6 +101,13 @@ spec:
       containers:
         - name: grader-setup-container
           image: {{ .Values.graderSetupService.graderSetupImage }}
+          resources:
+            requests:
+              cpu: {{ .Values.graderSetupService.graderCPU | default "0.5" }}
+              memory: {{ .Values.graderSetupService.graderMEM | default "1Gi" }}
+            limits:
+              cpu: {{ .Values.graderSetupService.graderCPU | default "0.5" }}
+              memory: {{ .Values.graderSetupService.graderMEM | default "1Gi" }}
```

Comment:
@Abhi94N this is a minor comment but not sure if having `requests` and `limits` set to the same value is worth it. In this case, we are telling the grader-setup-service to _have_ to use 0.5 CPU / 1g memory even if it's not required.

Reply:
@jgwerner updated repo and semver 5.1.0 to remove requests and only keep limits